src/main/java/com/birdie/kafka/connect/smt/DebeziumJsonDeserializer.java

```diff
@@ -121,6 +125,11 @@ public void configure(Map<String, ?> props) {
         unionPreviousMessagesSchemaLogUnionErrors = config.getBoolean(ConfigName.UNION_PREVIOUS_MESSAGES_SCHEMA_LOG_UNION_ERRORS);
         unionPreviousMessagesSchema = config.getBoolean(ConfigName.UNION_PREVIOUS_MESSAGES_SCHEMA);
         useProbabilisticFastPath = config.getBoolean(ConfigName.PROBABILISTIC_FAST_PATH);
+        List<String> ignoredFields = new ArrayList<String>();
+        ignoredFields = Arrays.asList(config.getString(ConfigName.IGNORED_FIELDS).split(","))
+                              .stream()
+                              .map(field -> field.replace('.', '_'))
```

Comment:
How does it work with fields that have a `_` in them? Can you add a test for it?

Reply:
Done :)
But that made me think that this doesn't take the topic into account if tables in the same connector have the same field, it will be ignored in both. Should we do something similar to what you did with the initial schema option?

Reply:
Great question. As far as I know we don't need it yet so I wouldn't yet :)